zrml/swaps/src/lib.rs

```diff
@@ -1256,7 +1256,6 @@ mod pallet {
         /// * `scoring_rule`: The scoring rule that's used to determine the asset prices.
         /// * `swap_fee`: The fee applied to each swap (mandatory if scoring rule is CPMM).
         /// * `weights`: These are the raw/denormalized weights (mandatory if scoring rule is CPMM).
-        #[frame_support::transactional]
```

Comment:
Why remove this? If we call this from an extrinsic in the prediction markets pallet, then we should make it so it reverts the entire logic

Reply:
Good catch, tbh. I have no idea how I managed to remove it, it was not on purpose.